docs/content/configuring-npm/package-json.md

```diff
@@ -341,9 +341,9 @@ install into the PATH. npm makes this pretty easy (in fact, it uses this
 feature to install the "npm" executable.)
 
 To use this, supply a `bin` field in your package.json which is a map of
-command name to local file name. On install, npm will symlink that file
-into `prefix/bin` for global installs, or `./node_modules/.bin/` for local
-installs.
+command name to local file name. When this packaged is installed, npm
+will symlink that file into a place where it can be invoked by name if
+installed globally, or by `npm exec` when installed in another package.
```

Comment:
```suggestion
installed globally, or by `npm exec` or `npm run-script` when installed in another package.
```
“when installed globally” is only true when the PATH contains the global npm bin location; should that be called out?

Reply:
The "source" of truth for executables is at https://docs.npmjs.com/cli/v7/configuring-npm/folders#executables so we should link there instead of trying to re-describe it.

Documenting "{prefix}/bin needs to be in your PATH" should probably be in the configuring section.

Reply:
Actually `run-script` is incorrect.  That only works for `scripts` entries, not `bin`.  Removing that.

Reply:
`npm run-script` entries in "another package" definitely has access to the bin tho; that's what i was trying to convey.

Reply:
Ah yeah, you can't `npm run-script` that package by name but you can refer to it by name in things run *via* `npm run-script`. I hope the new copy reflects this better.